definitions/npm/angular_v1.4.7/flow_v0.47.x-/angular_v1.4.7.js

```diff
@@ -7,9 +7,16 @@
 
 declare module angular {
 
+  declare type ScopeBindings = '<' | '=' | '&' | '<?' | '=?' | '&?' | '=ngModel';
+  declare type Scope = {[key: string]: ScopeBindings};
+  declare type ControllerFunction = (...a: Array<*>) => void;
+
   // I'm not sure how represent this properly: Angular DI declarations are a
   // list of strings with a single function at the end. The function can vary,
   // so it is a type param.
+  //
+  // NOTE: if you use compile step to mangle array, replace below with
+  // declare type $npm$angular$DependencyInjection<T> = T
```

Comment:
This is something i would like to be able to handle in better way than just writing a comment here. We don't use array dependency injection syntax with angular, and i think it's pretty common approach.